app/models/user.rb

```diff
@@ -1,2 +1,7 @@
 class User < ApplicationRecord
+  validates_presence_of :email
+
+  def flat_key
+    ENV['FLAT_KEY']
+  end
 end
```

Comment:
Need to replace hardcoding with OAuth token reference once that functionality is built out.